nodejs/streaming_pipe_in.mjs

```diff
@@ -0,0 +1,19 @@
+//Script reads text from a named pipe and writes it to stdout, capitalized
+
+import * as fs from 'fs';
+
+const pipeIn = process.argv[2];
+
+const input = fs.createReadStream(pipeIn);
+
+let remainingData = '';
+
+input.on('data', (chunk) => {
+  remainingData += chunk;
+  const lines = remainingData.split('\n');
+
+  for (let i = 0; i < lines.length - 1; i++) {
+    console.log(lines[i].toUpperCase());
+  }
+  remainingData = lines[lines.length - 1];
+});
```

Comment:
```suggestion
const rl = readline.createInterface({ input })

for await (const line of rl) {
  console.log(line.toUpperCase())
}
```

I was thinking more like this ☝️ . Can you make `streaming_pipe_in_and_out.mjs` follow this pattern, too?